File: java-breeds/app/src/main/java/es/tipolisto/breeds/data/database/AppDatabase.java
package es.tipolisto.breeds.data.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import es.tipolisto.breeds.data.model.RecordEntity;

@Database(entities = {RecordEntity.class}, version = 2)
public abstract class AppDatabase extends RoomDatabase {
    //Esto representa a la entidad o tabla usuarios
    public abstract RecordDao recordDao();
}

File: kotlin-breeds/app/src/main/java/es/tipolisto/breeds/data/database/AppDataBase.kt
package es.tipolisto.breeds.data.database

import androidx.room.Database
import androidx.room.RoomDatabase
import es.tipolisto.breeds.data.database.favorites.FavoritesDao
import es.tipolisto.breeds.data.database.favorites.FavoritesEntity
import es.tipolisto.breeds.data.database.records.RecordDao
import es.tipolisto.breeds.data.database.records.RecordEntity

@Database(entities = [RecordEntity::class, FavoritesEntity::class], version = 1)
abstract class AppDataBase : RoomDatabase() {
    abstract fun recordDao(): RecordDao
    abstract fun favoritesDao(): FavoritesDao
}